src/components/ConsoleComponent.js

```diff
@@ -7,16 +7,27 @@ const useStyles = makeStyles((theme) => ({
     padding: theme.spacing(2),
     color: theme.palette.common.white,
     background: theme.palette.common.black,
-    height: '100%',
+    height: '200%',
```

Comment:
Why did you need to change this height? Just curious!

Reply:
I just felt it looked better, without the increased height the console can be a bit narrow and you can't see entirely the box. However I can totally change it back if we would prefer it fit on one page

Reply:
I think maybe ideally everything would fit on one page without needing to scroll, but it needed to scroll before anyway, and I think you're right, this is better for now at least because you can see more of the console!